frontend/i18n/pl/projects/overview.js

```diff
@@ -20,5 +20,18 @@ export default {
     'Mowa na tekst',
     'Anotacja do artykułu',
     'Anotacja afektywna'
-  ]
+  ],
+  tags: 'Tagi',
+  allowSingleLabel: 'Pozwól na pojedynczą etykietę',
+  allowOverlapping: 'Pozwól na nakładanie się encji',
+  useRelation: 'Użyj etykietki relacji',
+  count: 'Licz',
+  graphemeClusters: 'klastry grafemowe',
+  emojiDescription: 'Jak emoji(🌷, 💩, i 👍), CRLF(i tak dalej)',
+  asOneCharacter: 'jako jeden znak',
+  enableSummary: 'Umożliwić użytkownikowi dodawanie podsumowania',
```

Comment:
I haven't taken more detailed look but I have a question about the translations... in the requirements, it was mentioned to do the UI in polish but the options in the project creation page should be in english. So I'm not sure if this is necessary. But maybe we can keep it for now? What do you think @haanh764 ?

Reply:
yeah I remember he said that the creation page could be in English, only the UI of annotation must be in Polish, but I think because we translated everything into multiple languages, so maybe just keep it for now to keep the consistencies. Because it'll be a bit weird if we have mixed Eng and Pol together :sweat_smile:

Reply:
Yeah, I was thinking because sometimes some words and phrases are not really meant to be translated because they will sound strange/funny in the other language, and maybe it was the reason why the original doccano didn't have translations for some strings like the relational labelling etc (and why they didn't want us to make these options in Polish). But perhaps I am just being too paranoid. 🤣 Okay, let's keep them XD

Reply:
Yeah, I was thinking of inconsistencies which is why I tried to translate all of them, but probably some of the words can be left in English XD   anyway this part is for admin only so I think it won't matter to the annotator, but if you think that keeping them in English is better I can remove the translations XD